frontend/package.json

```diff
@@ -96,6 +96,7 @@
     "lodash-es": "^4.17.21",
     "lucide-react": "^0.338.0",
     "lz-string": "^1.5.0",
+    "mathjs": "^12.4.2",
```

Comment:
i think youll want to run `pnpm install` in `cd frontend`, and commit the changes to the lockfile

Reply:
maybe can remove this with the suggestion below

Reply:
I've updated the lockfile, but don't know whether it's right